### PR TITLE
add qnap entries to existing section (`myqnapcloud.cn` , `mycloudnas.com`, `mynascloud.com`)

### PR DESCRIPTION
**Pull request by qnapnickchang**
add a new domain for our public dynamic dns service

Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

Organization Website: https://www.qnap.com
QNAP System Inc. is a corporation that specializes in Network-attached storage (NAS) appliances.

Reason for PSL Inclusion
====

Hello, I'm engineer from qnap.com and we would like to add a new domain for our public dynamic dns service. 
Number of users this request is being made to serve: 1,00k

DNS Verification via dig
=======
```
dig +short TXT _psl.myqnapcloud.cn
"https://github.com/publicsuffix/list/pull/1837"
```

```
dig +short TXT _psl.mycloudnas.com
"https://github.com/publicsuffix/list/pull/1837"
```

```
dig +short TXT _psl.mynascloud.com
"https://github.com/publicsuffix/list/pull/1837"
```

Results of Syntax Checker (`make test`)
=========

### CODE REVIEW

**Review by simon-friedberger — CHANGES_REQUESTED** [review]
Please fix the sorting. (The old entries are also wrong.) If you can, please add a non-personal e-mail.

**Comment by simon-friedberger** [comment]
@qnapnickchang Any updates?

**Comment by qnapnickchang** [comment]
Hi @simon-friedberger  fix soring. please check again. thanks

**Comment by simon-friedberger** [comment]
* [ ] Expiration (Note: Must STAY >2y at all times)
  * [x] mycloudnas.com expires 2027-07-19
  * [ ] mynascloud.com expires 2025-08-19
  * [x] myqnapcloud.cn expires 2026-07-10
  * [ ] dev-myqnapcloud.com expires 2024-08-16
  * [x] alpha-myqnapcloud.com expires 2027-02-23
  * [x] myqnapcloud.com expires 2027-01-24
* [x] DNS _psl entries (Note: Must STAY in place)
  * [x] mycloudnas.com
  * [x] mynascloud.com
  * [x] myqnapcloud.cn
  * [x] dev-myqnapcloud.com
  * [x] alpha-myqnapcloud.com
  * [x] myqnapcloud.com
* [x] Tests pass
* [ ] Sorting
* [x] Reasoning/Organization description

**Comment by simon-friedberger** [comment]
https://github.com/publicsuffix/list/wiki/Guidelines#sort-your-submission-correctly-important

**Comment by qnapnickchang** [comment]
Hi @simon-friedberger 

- domain renew. Please check again.
- Fix soring 

Thanks

File: public_suffix_list.dat [hunk 1]
```diff
@@ -14138,8 +14138,11 @@ qcx.io
 
 // QNAP System Inc : https://www.qnap.com
 // Submitted by Nick Chang <nickchang@qnap.com>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:14140]: Please use an address which isn't tied to a specific person.

**Comment by qnapnickchang** [comment]
@simon-friedberger 
Updated. Please check again.

Thanks

**Pull request merged**